spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/GovernanceHeaderStatusUtils.java

```diff
@@ -17,13 +17,17 @@
 
 package com.huaweicloud.governance.adapters;
 
+import java.util.Random;
+
 import org.springframework.core.env.Environment;
 
 public class GovernanceHeaderStatusUtils {
   private static final String RESPONSE_STATUS_HEADER_DEFAULT_KEY = "X-HTTP-STATUS-CODE";
 
   public static final String HEADER_KEY = "spring.cloud.servicecomb.governance.response.header.status.key";
 
+  public static final Random random = new Random();

```

Comment:
Do not use global variable, and Random is not thread safe.

Reply:
fixed